src/global/global_init.cc

```diff
@@ -136,14 +136,14 @@ void global_pre_init(
 				    &cerr, flags);
   if (ret == -EDOM) {
     cct->_log->flush();
-    cerr << "global_init: error parsing config file." << std::endl;
+    cerr << "global_pre_init: error parsing config file." << std::endl;
     _exit(1);
```

Comment:
Why not use `__func__` as name field. That'll fix it for once and forever, even if the name changes.